2-Intelligent-Agents/cleaningRobot.js

```diff
@@ -33,9 +33,10 @@ class World {
 
 // Rules are defined in code
 function reflexVacuumAgent(world) {
-    if (world.floors[world.location].dirty) { return 'SUCK'; }
+    if (world.floors[world.location].dirty) { return 'SUCK'; }                                                                
     else if (world.location == 0)           { return 'RIGHT'; }
-    else if (world.location == 1)           { return 'LEFT'; }
+    else if (world.location == 1)           { return'LEFT'; }
```

Comment:
I don't understand what you're trying to do with the change to this function

Reply:
It was a mistake I might have committed when I was working to implement the visualization.
I will correct this and all the above mentioned.
Thanks for the PR.